cooltools/api/dotfinder.py

```diff
@@ -1569,8 +1416,7 @@ def scoring_and_extraction_step(
         verbose=very_verbose,
     )
 
-    # composing/piping scoring and histogramming
-    # together :
+    # composing/piping scoring and histogramming together :
     job = lambda tile: to_extract(to_score(tile))
```

Comment:
do I understand correcdtly that the hists are just per tile?

Reply:
yeah - because bins are pre-defined (lambda-chunks) we just built "partial"-histograms per tile and then aggregate them together. We cannot have a shared global histogram for several processes to write to - because - Python ...